packages/terra-abstract-modal/package.json

```diff
@@ -30,11 +30,11 @@
   },
   "dependencies": {
     "classnames": "^2.2.5",
-    "focus-trap-react": "^4.0.1",
     "keycode-js": "^1.0.4",
     "prop-types": "^15.5.8",
     "react-portal": "^4.1.2",
-    "terra-doc-template": "^2.2.0"
+    "terra-doc-template": "^2.2.0",
+    "ally.js": "^1.4.1"
```

Comment:
ally.js doesn't seem to be actively maintained: https://github.com/medialize/ally.js. Do we have any other options from a dependency standpoint?

Reply:
I tested https://github.com/WICG/inert but ran into issues with IE10. I believe that IE10 is not full supported by that polyfill, but we may be able to fork that and add IE10 support.

I also tested https://github.com/GoogleChrome/inert-polyfill and it worked in Chrome, Firefox, and Safari but I wasn't able to get it to work in IE.

I settled on https://github.com/medialize/ally.js as it provided the inert functionality and worked in IE10, IE11, Chrome, Firefox, and Safari.

Reply:
I spent some more time on this and was able to get https://github.com/WICG/inert working in IE10 with the addition of a MutationObserver polyfill: https://www.npmjs.com/package/mutationobserver-shim

https://github.com/WICG/inert was last updated March 6th, 2019
https://www.npmjs.com/package/mutationobserver-shim was last updated January 15th, 2019

Commit: https://github.com/cerner/terra-framework/pull/608/commits/27441f14ebccf37089a9d5fe5ac27e0849a853fd

Reply:
With switching to https://github.com/WICG/inert, Jest/JSDom has some issues with it. I've [logged an issue to Jest](https://github.com/facebook/jest/issues/8373) to see if that team can provide some guidance. 

For now, I'm only requiring the inert polyfill if the node.env is not set to `test` which is true when running Jest tests: https://github.com/cerner/terra-framework/pull/608/commits/cb0df478f02a3e80b27f18e66caf5bea9d0f263c